timestring/Date.py

```diff
@@ -28,6 +28,23 @@
     mon=1, tue=2, tues=2, wed=3, wedn=3, thu=4, thur=4, fri=5, sat=6, sun=7,
     mo=1, tu=2, we=3, th=4, fr=5, sa=6, su=7,
 )
+RELATIVE_DAYS = {
+    'now': 0,
+    'today': 0,
+    'yesterday': -1,
+    'tomorrow': 1,
+    'day before yesterday': -2,
+    'day after tomorrow': 2,
+}
+DAYTIMES = dict(
+    morning=9,
```

Comment:
let's make this 4

Reply:
This daytime was already defined. We will change it in a separate PR because it is not related to the changes in the PR.